apio-consumer/src/debug/kotlin/com/liferay/apio/consumer/IdlingResources.kt

```diff
@@ -12,8 +12,8 @@ object IdlingResources {
 	}
 
 	fun unregisterOkHttp(okHttpClient: OkHttpClient, name: String) {
-//		val create = OkHttp3IdlingResource.create("_", okHttpClient)
-//		IdlingRegistry.getInstance().unregister(create)
+		val create = OkHttp3IdlingResource.create("_", okHttpClient)
```

Comment:
Is this method called now?

Reply:
I don't think so, they are unused in the project. So I just removed them, it's better than keep it commented. Ok?